tests/system/providers/ftp/example_ftp.py

```diff
@@ -57,7 +61,30 @@
     )
     # [END howto_operator_ftp_get]
 
+    # [START howto_operator_ftps_put]
+    ftps_put = FTPSFileTransmitOperator(
+        task_id="test_ftps_put",
+        ftp_conn_id="ftps_default",
+        local_filepath="/tmp/filepath",
+        remote_filepath="/remote_tmp/filepath",
+        operation=FTPOperation.PUT,
```

Comment:
Just want to double check this is intentional and not supposed to be `FTPSOperation.PUT`

Reply:
Yes that is intentional, as FTPS still uses the same operations (PUT and GET) as FTPFileTransmitOperator.

Reply:
I think it should be alright.